lib/tailwind_form_builder.rb

```diff
@@ -0,0 +1,78 @@
+class TailwindFormBuilder < ActionView::Helpers::FormBuilder
+  include ActionView::Helpers::TagHelper
+
+  class << self
+    def has_styles_for(fields, style)
+      @styles_for ||= {}
+
+      fields = Array(fields)
+      fields.each do |field|
+        @styles_for[field] = style
+      end
+    end
+
+    def styles_for
+      @styles_for ||= {}
+    end
+  end
+
+  (field_helpers - [ :label, :check_box, :radio_button, :fields_for, :fields, :hidden_field, :file_field ]).each do |selector|
+    class_eval <<-RUBY_EVAL, __FILE__, __LINE__ + 1
+      def #{selector}(method, options = {})
+        merged_opts = apply_classes(options, self.class.styles_for[:#{selector}])
+        super(method, merged_opts)
+      end
+    RUBY_EVAL
+  end
```

Comment:
Could we not have all the styles/config in this file rather than spreading over two?